src/main/java/de/retest/recheck/review/ignore/PixelDiffFilter.java

```diff
@@ -15,6 +18,7 @@
 @Getter
 @Slf4j
 public class PixelDiffFilter implements Filter {
+	//TODO Move this web specific class to recheck-web
```

Comment:
As it belongs to the class, I suggest to move it above the annotations. (And please insert a space: `// TODO ...` 🤓)